TSError: ⨯ Unable to compile TypeScript

Error on startup:
```
message-bus-tester_1    | /service/message-bus-tester/node_modules/ts-node/src/index.ts:245
message-bus-tester_1    |     return new TSError(diagnosticText, diagnosticCodes)
message-bus-tester_1    |            ^
message-bus-tester_1    | TSError: ⨯ Unable to compile TypeScript:
message-bus-tester_1    | src/main.ts(4,34): error TS2305: Module '"../node_modules/@libero/event-bus/dist"' has no exported member 'RabbitMessageQueue'.
message-bus-tester_1    | src/main.ts(4,54): error TS2305: Module '"../node_modules/@libero/event-bus/dist"' has no exported member 'MockMessageQueue'.
message-bus-tester_1    | 
message-bus-tester_1    |     at createTSError (/service/message-bus-tester/node_modules/ts-node/src/index.ts:245:12)
message-bus-tester_1    |     at reportTSError (/service/message-bus-tester/node_modules/ts-node/src/index.ts:249:19)
message-bus-tester_1    |     at getOutput (/service/message-bus-tester/node_modules/ts-node/src/index.ts:362:34)
message-bus-tester_1    |     at Object.compile (/service/message-bus-tester/node_modules/ts-node/src/index.ts:395:32)
message-bus-tester_1    |     at Module.m._compile (/service/message-bus-tester/node_modules/ts-node/src/index.ts:473:43)
message-bus-tester_1    |     at Module._extensions..js (internal/modules/cjs/loader.js:700:10)
message-bus-tester_1    |     at Object.require.extensions.(anonymous function) [as .ts] (/service/message-bus-tester/node_modules/ts-node/src/index.ts:476:12)
message-bus-tester_1    |     at Module.load (internal/modules/cjs/loader.js:599:32)
message-bus-tester_1    |     at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
message-bus-tester_1    |     at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

--- a/message-bus-tester/src/main.ts
+++ b/message-bus-tester/src/main.ts
@@ -1,7 +1,7 @@
 import * as express from "express";
 import { Request, Response } from "express";
 import { InfraLogger as logger } from './logger';
-import { Event, EventIdentifier, RabbitMessageQueue, MockMessageQueue } from '@libero/event-bus';
+import { Event, EventIdentifier, RabbitEventBus, MockEventBus } from '@libero/event-bus';
 
 type TestEventPayload = {
   x: number;
@@ -15,7 +15,7 @@ const init_mq = async () => {
   };
 
   // Mock bus
-  const mock_mq = await (new MockMessageQueue()).init([test_event_def, test_event_def], "message-bus-test");
+  const mock_mq = await (new MockEventBus()).init([test_event_def, test_event_def], "message-bus-test");
   logger.info("messageQueueStarted");
 
   mock_mq.subscribe<TestEventPayload>(test_event_def, async (event) => {
@@ -38,7 +38,7 @@ const init_mq = async () => {
   }, 10000)
 
   // Rabbit bus
-  const rabbitmq_mq = await (new RabbitMessageQueue()).init([test_event_def, test_event_def], "message-bus-test");
+  const rabbitmq_mq = await (new RabbitEventBus()).init([test_event_def, test_event_def], "message-bus-test");
   logger.info("messageQueueStarted");
 
   rabbitmq_mq.subscribe<TestEventPayload>(test_event_def, async (event) => {

